cortex-mixin/dashboards/writes.libsonnet

```
@@ -204,34 +240,70 @@ local utils = import 'mixin-utils/utils.libsonnet';
           'Compactions / sec',
           'sum(rate(cortex_ingester_tsdb_compactions_total{%s}[$__rate_interval]))' % [$.jobMatcher($._config.job_names.ingester)],
           'sum(rate(cortex_ingester_tsdb_compactions_failed_total{%s}[$__rate_interval]))' % $.jobMatcher($._config.job_names.ingester),
+        ) +
+        $.panelDescription(
+          'Compactions / sec',
+          |||
+            Ingesters maintain a local TSDB per-tenant on disk. Each TSDB maintains a head block for each
+            active time series; these blocks get periodically compacted (by default, every 2h).
+            This panel shows the rate of compaction operations across all TSDBs on all ingesters. 
+          |||
         ),
       )
       .addPanel(
         $.panel('Compactions latency') +
-        $.latencyPanel('cortex_ingester_tsdb_compaction_duration_seconds', '{%s}' % $.jobMatcher($._config.job_names.ingester)),
+        $.latencyPanel('cortex_ingester_tsdb_compaction_duration_seconds', '{%s}' % $.jobMatcher($._config.job_names.ingester)) +
+        $.panelDescription(
+          'Compaction Latency',
+          |||
+            The average, median (50th percentile), and 99th percentile time ingesters take to compact head blocks
+            on the local filesystem.
+          |||
+        ),
       )
     )
     .addRowIf(
       std.member($._config.storage_engine, 'blocks'),
-      $.row('Ingester - Blocks storage - TSDB WAL')
+      $.row('Ingester - Blocks storage - TSDB Write Ahead Log (WAL)')
```

```suggestion
      $.row('Ingester - blocks storage - TSDB write ahead log (WAL)')
```
Where does this display to the user?